ctk/common/src/main/kotlin/org/codice/compliance/verification/binding/RedirectBindingVerifier.kt

```diff
@@ -205,7 +207,7 @@ class RedirectBindingVerifier(httpResponse: Response) : BindingVerifier(httpResp
                 "encoding currently.")
 
         decodedMessage.debugPrettyPrintXml("Decoded SAML Response")
-        return Common.buildDom(decodedMessage)
+        return decodedMessage
```

Comment:
✏️ Is there a reason we're not verifying the signature inside this method like in PostBindingVerifier?

Reply:
I forgot to change the post one. I didn't want to hide this in the decode. That way the decode is doing more than what the name says.